AssetTools.NET/Extra/MonoDeserializer/MonoCecilTempGenerator.cs

```diff
@@ -55,7 +55,7 @@ public List<AssetTypeTemplateField> Read(AssemblyDefinition assembly, string nam
         {
             this.unityVersion = unityVersion;
```

Comment:
The Unity version is set here, so why do you pass it to all of the other methods?

Reply:
Oh, missed that, since the version wasn't set in constructor